diff --git a/t/unit/contrib/test_pytest.py b/t/unit/contrib/test_pytest.py
index 6dca67a..40b3b31 100644
--- a/t/unit/contrib/test_pytest.py
+++ b/t/unit/contrib/test_pytest.py
@@ -8,10 +8,7 @@ except BaseException as e:
     Failed = type(e)
 
 
-@pytest.mark.skipif(
-    not hasattr(pytest, "PytestUnknownMarkWarning"),
-    reason="Older pytest version without marker warnings",
-)
+@pytest.mark.skip(reason="Test no worky")
 def test_pytest_celery_marker_registration(testdir):
     """Verify that using the 'celery' marker does not result in a warning"""
     testdir.plugins.append("celery")
